src/gradient/gradient.class.ts

```diff
@@ -93,7 +90,7 @@ export class Gradient<
     gradientTransform,
     id,
   }: GradientOptions<T>) {
-    const uid = fabric.Object.__uid++;
+    const uid = FabricObject.__uid++;
```

Comment:
this is not the place for the __uid. i remember @ShaMan123 already worked to put it somewhere else